src/main/resources/com/google/api/codegen/java/message.snip

```diff
@@ -112,7 +112,7 @@
   }
   private static final {@schema.innerTypeName} DEFAULT_INSTANCE;
   static {
-    DEFAULT_INSTANCE = new {@schema.innerTypeName}();
+      DEFAULT_INSTANCE = new {@schema.innerTypeName}();
```

Comment:
So what should be done to fix this? Without the new extra indentation, the resulting view doesn't have an indentation.